lib/reporters/ConsoleReporter.js

```diff
@@ -0,0 +1,37 @@
+// TODO: finish grouping once suiteStart is implemented
+var hasGrouping = "group" in console && "groupEnd" in console;
+
+export default class ConsoleReporter {
+    constructor(runner) {
+        runner.on("suiteStart", this.onSuiteStart.bind(this));
```

Comment:
Looks like the bind calls aren't needed here, as there is no access to `this` inside any of the methods.